doc/decoding/decoding_tutorial.rst

```diff
@@ -83,14 +85,14 @@ The ``haxby_dataset`` object has several entries that contain paths to the files
 downloaded on the disk::
 
     >>> print(haxby_dataset)  # doctest: +SKIP
-    {'anat': ['/home/varoquau/dev/nilearn/nilearn_data/haxby2001/subj1/anat.nii.gz'],
-    'func': ['/home/varoquau/dev/nilearn/nilearn_data/haxby2001/subj1/bold.nii.gz'],
-    'mask_face': ['/home/varoquau/dev/nilearn/nilearn_data/haxby2001/subj1/mask8b_face_vt.nii.gz'],
-    'mask_face_little': ['/home/varoquau/dev/nilearn/nilearn_data/haxby2001/subj1/mask8_face_vt.nii.gz'],
-    'mask_house': ['/home/varoquau/dev/nilearn/nilearn_data/haxby2001/subj1/mask8b_house_vt.nii.gz'],
-    'mask_house_little': ['/home/varoquau/dev/nilearn/nilearn_data/haxby2001/subj1/mask8_house_vt.nii.gz'],
-    'mask_vt': ['/home/varoquau/dev/nilearn/nilearn_data/haxby2001/subj1/mask4_vt.nii.gz'],
-    'session_target': ['/home/varoquau/dev/nilearn/nilearn_data/haxby2001/subj1/labels.txt']}
+    {'anat': ['/home/user/nilearn_data/haxby2001/subj1/anat.nii.gz'],
```

Comment:
Well it is still a *"fake"* display, so it won't really change anything.
On Python, $USER has no meaning by the way, I don't think it improves readability (which was the only reason for my change `varoquau/dev/nilearn` → `user`).